src/Tizen.NUI/src/public/BaseComponents/ViewAccessibility.cs

```diff
@@ -0,0 +1,271 @@
+/*
+ * Copyright(c) 2021 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+using Tizen.NUI;
+
+namespace Tizen.NUI.BaseComponents
+{
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class Address {
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Address(string xbus, string xpath) {
+            bus = xbus;
+            path = xpath;
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string bus { get; set; }
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string path { get; set; }
+    }
+
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class AddressCollection : SafeHandle {
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public AddressCollection(IntPtr ptr) : base(ptr, true) {}
```

Comment:
This code is build warning-CA1720:Identifier 'ptr' contains type name	Tizen.NUI	
https://docs.microsoft.com/ko-kr/dotnet/fundamentals/code-analysis/quality-rules/ca1720

Reply:
Done.